lib/miq_automation_engine/engine/miq_ae_engine/miq_ae_method.rb

```diff
@@ -228,8 +228,8 @@ def self.run_method(cmd)
         msg = "Method exited with rc=#{verbose_rc(rc)}"
         method_pid = nil
         threads = []
-      rescue => err
-        STDERR.puts "** AUTOMATE ** Method exec failed because #{err.class}:#{err.message}" if ENV.key?("CI")
+      rescue StandardError => err
+        warn "** AUTOMATE ** Method exec failed because #{err.class}:#{err.message}" if ENV.key?("CI")
```

Comment:
@mkanoor Do we want to keep this one as a `STDERR.puts`?  It was added in https://github.com/ManageIQ/manageiq/pull/11627 to help debug a sporadic Travis failure.  Not sure if it is still useful.

Reply:
@gmcculloug Yes we need that message because we need to debug travis failures. Previously when we used to see intermittent failures we could never diagnose why a method failed.

Reply:
@mkanoor 

So we will keep the STDERR.puts and put a # rubocop:disable Style/StderrPuts.

I am wondering Is that because we don't want the error be disabled? https://github.com/rubocop-hq/rubocop/issues/4888

Reply:
@astrozzc That's not exactly what I was thinking about when I suggested reverting it, but reading more into the issue we do want to avoid the situation where `warn` could be disabled.